core/README.md

```diff
@@ -0,0 +1,160 @@
+# `@commercetools-test-data/core`
+
+This package provides the core functions to define the data models.
+
+## Install
+
+```bash
+$ yarn add -D @commercetools-test-data/core
+```
+
+## Creating a new data model
+
+Models are treated packages. Each model will be published as a standalone NPM package.
```

Comment:
What's the rationale behind defining the models as standalone NPM packages?

Reply:
Just so that you can pick what you want/need.

I suppose we can maybe group the models by domain or something instead.

- `commons`: for things like https://docs.commercetools.com/api/types
- `products`: for models related to the product domain.
- `accounts`: things like user, project, etc.

Would that be reasonable? cc @tdeekens

Reply:
I like the idea but wonder it it's generally that useful however does not hurt. If we group by domain it might be helpful. You could ask the same for babel in parts I suppose.